src/01-general/invites.apib

```diff
@@ -0,0 +1,34 @@
+## Invites [/invites]
+
+*Required scopes: `users`*
+
+### invites.list [GET /invites.list]
```

Comment:
Q: Should we make this endpoint more specific? IMHO `invite` is quite generic and since this is coupled to the user creation flow I would even suggest something link `/users.invites` for the list and `/users.cancelInvite` to cancel a specific invite? Thoughts?

Reply:
An invite is not a `/users` resource and thus has it's own resource endpoint `/invites` An invite is not a user in any way.

Reply:
Indeed, but the invite is for a user, would `/user-invites.list` make more sense then? Because only `/invite` is too generic IMHO.

Reply:
I'm actually against `users.invites` (because I want to avoid subresources), but no strong opinion on `invites` vs `userInvites` (mind the camelCasing).

Reply:
No strong opinion here as well. `userInvites` would indeed make it more explicit what kind of invite we're talking about. Although I don't think we don't have any other invitation system? Although once we start working with inviting remote people to calendar events this might become a bit more confusing. So in favor of `userInvites` :ok_hand:

Reply:
I believe we would be safer with `userInvites` as well.

Reply:
I don't see the benefit of calling it `userInvite` ... If you start adding remote people for the calendar, then you are also inviting users there. So it won't be better in that case, right?

I personally would stick with invites here and then go for a more accurate solution for inviting people to events when we actually start doing it.

(An option, for example, would be to use the same endpoint for both because the logic on the API level will be the same)

Reply:
I was also going to suggest going for this name for now. But because we already saw that it could cause conflicts in other domains of our application I suggest to rename it now rather than having to introduce a BC breaking change later. There's no downside to renaming it, only making it more explicit.

Reply:
But where is userInvites more explicit that just invites? `userInvites` could also be the invites endpoint for `inviting users to events`, right?

Reply:
Inviting someone to an event is a different action with different side effects and maybe even other required properties for sending the invite. So I would not mix them together, that would only make things very confusing for third party developers.

Reply:
I was saying that `userInvites` could also be understood as the endpoint for `inviting users to events` ... So that it is not clear either ...

So if we say invites is not clear enough, then `userInvites` isn't either... We should then think about a more clear name

Reply:
Should we maybe take this discussion offline? There are some different opinions here and commenting back and forth won't really help us here I'd say.

Reply:
We discussed this offline and the decision was to make it `/userInvites.*`